channels/packages/node-manager/develop/deployment-storageos-node-manager.yaml

```diff
@@ -64,7 +69,7 @@ spec:
       priorityClassName: system-cluster-critical
       restartPolicy: Always
       securityContext:
-        runAsNonRoot: true
+        runAsUser: 65532
```

Comment:
What's the reason for this?

Reply:
This change is from an old PR that has been merged already https://github.com/storageos/operator/pull/156
It's here due to a rebase, this branch was created before it